lcd_test/lcd_test.go

```diff
@@ -1,4 +1,4 @@
-package lcd_test
+package lcdtest
```

Comment:
nit, what if we rename this package to `api` or `rest` instead of `lcd_test`? It'd be much more straightforward to devs that are not familiar with the terminology

Reply:
not blocking though

Reply:
I'd actually propose to rename the whole directory/package to something like `test`, `tests`, or `testing`